Update getting started docs for new users (#1330)
ESMValGroup/ESMValTool: doc/sphinx/source/getting_started/running.rst

is available in the ESMValTool installation folder as
``examples/recipe_python.yml``.

This recipe finds data from CanESM2 and MPI-ESM-LR for 2000 - 2002,
extracts a single level (850 hPa), regrids it to a 1x1 degree mesh and runs
a diagnostic script that creates some plots of Air temperature and
precipitation flux. You can download the recipe from
`github <https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/recipes/examples/recipe_python.yml>`_
and save it in your project directory as (e.g.) ``recipe_python.yml``
and then run ESMValTool with

.. code:: bash

	esmvaltool -c /path/to/config-user.yml recipe_python.yml --synda

The ``--synda`` option tells ESMValTool to use Synda to search for and download
the necessary datasets.

ESMValTool will also find recipes that are stored in its installation directory.
A copy of the example recipe is available in
``/path/to/installation/esmvaltool/recipes/examples/recipe_python``.
Thus, the following also works:

.. code:: bash

	esmvaltool -c /path/to/config-user.yml examples/recipe_python.yml

Note that this command does not call Synda. The required data should thus be

> located in the directories specified in your user config file.

Reviewer: ```suggestion
located in the directories specified in your user configuration file.
```

> Recall that the chapter :ref:`User configuration file <config-user>`

Reviewer: ```suggestion
Recall that the chapter :ref:`Configuring ESMValTool <config-user>`
```
since you renamed the chapter

Running ESMValTool
******************

ESMValTool is mostly used as a command line tool. Whenever your
conda environment for ESMValTool is active, you can just run the command
``esmvaltool <options> <recipe>``. One of the options that must be specified

> is the user-config file, which is specified using the

Reviewer: ```suggestion
is the user configuration file, which is specified using the
```

option ``-c /path/to/config-user.yaml``. An example recipe is available in the

> ESMValTool installation folder as ``examples/recipe_python.yml``:

Reviewer: Make this a link to https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/recipes/examples/recipe_python.yml

	    - schlund_manuel

	  references:
	    - acknow_project

	  projects:
	    - esmval
	    - c3s-magic

	datasets:
	  - {dataset: CanESM2,  project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
	  - {dataset: MPI-ESM-LR,  project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}

	preprocessors:

	  preprocessor1:
	    extract_levels:
	      levels: 85000
	      scheme: nearest
	    regrid:
	      target_grid: 1x1
	      scheme: linear
	    multi_model_statistics:
	      span: overlap
	      statistics: [mean, median]

	diagnostics:

	  diagnostic1:
	    description: Air temperature and precipitation Python tutorial diagnostic.
	    themes:
	      - phys
	    realms:
	      - atmos
	    variables:
	      ta:
	        preprocessor: preprocessor1
	      pr:
	    scripts:
	      script1:
	        script: examples/diagnostic.py
	        quickplot:
	          plot_type: pcolormesh

This recipe finds data from CanESM2 and MPI-ESM-LR for 2000 - 2002,
extracts a single level (850 hPa), regrids it to a 1x1 degree mesh and runs
a diagnostic script that creates some plots of Air temperature and

> precipitation flux. You can copy the recipe above and save it in your project

Reviewer: ```suggestion
precipitation flux. You can copy the recipe above and save it in your project
```
```suggestion
precipitation flux. You can download the recipe above from [add link to https://github.com/ESMValGroup/ESMValTool/blob/version2_development/esmvaltool/recipes/examples/recipe_python.yml] and save it in your project
```

directory as (e.g.) ``example_recipe_python.yml`` and then run ESMValTool with

.. code:: bash

> 	esmvaltool -c /path/to/config-user.yml example recipe_python.yml -s

Reviewer: ```suggestion
	esmvaltool -c /path/to/config-user.yml recipe_python.yml --synda
```

> The ``-s`` option tells ESMValTool to use Synda to search for and download

Reviewer: ```suggestion
The ``--synda`` option tells ESMValTool to use Synda to search for and download
```

the necessary datasets.

> ESMValTool will also find recipes that are stored in its install directory.

Reviewer: ```suggestion
ESMValTool will also find recipes that are stored in its installation directory.
```

> A copy of the example recipe is available in ``examples/recipe_python``. Thus,

Reviewer: No, a copy is available in /path/to/installation/esmvaltool/recipes
